### DISCUSSION TITLE
if you need it as an API group by are [Aug 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6175] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **victorminemu**  
#### But it can be done client side  
#  
> 🙋‍♂️ **victorminemu**  
#### The problem that I see is the pagination  
#  
> 💁 **o1lab**  
#### How many records do you have in total  
#  
> 🙋‍♂️ **victorminemu**  
#### The table with more records that I have is about 130000  
#  
> 🙋‍♂️ **victorminemu**  
#### Maybe with webassembly we can speed up the grouping on the client side.  
#  
> 🙋‍♂️ **victorminemu**  
#### https://www.assemblyscript.org/  
#  
> 🙋‍♂️ **victorminemu**  
#### https://www.assemblyscript.org/compiler.html#programmatic-usage  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1126804084247232514)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
